assets/js/handlebars/templates/search-results.handlebars

```diff
@@ -36,11 +36,11 @@
   <p>The search functionality is full-text based. Here are some tips:</p>
 
   <ul>
-    <li>Multiple words (such as <code>foo bar</code>) are searched as <code>OR</code></li>
+    <li>Multiple words (such as <code>foo bar</code>) are searched as <code>AND</code></li>
     <li>Use <code>*</code> anywhere (such as <code>fo*</code>) as wildcard</li>
-    <li>Use <code>+</code> before a word (such as <code>+foo</code>) to make its presence required</li>
+    <li>Use <code>+</code> before a word to make its presence optional. For example <code>+this +or_that </code></li>
```

Comment:
It is a bit weird to use `+` for optional but I don't think we have another option, right? I am sorry for the back and forth on this but it may be better to just tell people to use `+type:function`. Unsure.

Reply:
In any case, it is a matter of picking the first commit or both. So just let me know your final thoughts and I will take it from here. Thank you.

Reply:
I think it is a bit weird, yes :) however, I think it's a bit weird either way honestly. It's an unconventional search experience regardless. I think I'm partial to making the more useful behavior the default (AND) especially given how unlikely I think users are to need `OR`. I understand the reservations though, whatever you decide will work for me in the end. As part of our (slow) migration back to hex I'll also be adding a "how to use this documentation" guide.

Reply:
Will do 👍

Reply:
Poll is here: https://twitter.com/ZachSDaniel1/status/1701590059391070711

It looks like it's relatively split, but leaning towards the new behavior introduced here. Its up for a week, I'm in no rush :)

Reply:
Sweet, thank you! Wake me up when ~september~ the pool ends.

Reply:
Okay, so it's still running, but it is almost *exactly* split which is...wild. I don't really care either way 😆. Probably better to err on the side of no change? The first commit to me is more important, allowing to filter on type. We're adding custom stuff to the search items with `type:dsl`, so that will help users search DSLs which is one of the main use cases of ash_hq.

Reply:
Yeah, I think no change and no additional code is best at the end.